### PR TITLE
Issue323 Fully implement attachment restapi endpoint

### PR DESCRIPTION
**Pull request by kovax**

### CODE REVIEW

File: restapi/src/main/java/org/cristalise/restapi/ItemOutcomeAttachment.java [hunk 1]
```diff
@@ -27,9 +27,17 @@
 import javax.ws.rs.Path;
```
Comment (kovax) [inline restapi/src/main/java/org/cristalise/restapi/ItemOutcomeAttachment.java]: Restapi test is missing. Workflow with 2 activities: first to save a jpeg, second to save a pdf to test MIME type handlign as well.

Comment (taurus227) [reply restapi/src/main/java/org/cristalise/restapi/ItemOutcomeAttachment.java]: Can the same activity save two different mime types?

Comment (kovax) [reply restapi/src/main/java/org/cristalise/restapi/ItemOutcomeAttachment.java]: hm, I just had a look, it is a kind of undefined. Currently there is a `AttachmentMimeTypes` Activity property, which contains the comma separated list of mine types.

Comment (kovax) [reply restapi/src/main/java/org/cristalise/restapi/ItemOutcomeAttachment.java:27]: yes, the mime type can be different for each activity/outcome version

**Pull request merged**